src/app/client/src/app/modules/groups/components/add-member/add-member.component.ts

```diff
@@ -44,11 +52,24 @@ export class AddMemberComponent implements OnInit, OnDestroy {
     if (!this.groupData) {
       this.location.back();
     }
+    this.initRecaptcha();
     this.instance = _.upperCase(this.resourceService.instance);
     this.membersList = this.groupsService.addFieldsToMember(_.get(this.groupData, 'members'));
     this.telemetryImpression = this.groupService.getImpressionObject(this.activatedRoute.snapshot, this.router.url);
   }
 
+  initRecaptcha() {
+    this.groupService.getRecaptchaSettings().subscribe((res: any) => {
+      if (res.result.response) {
+        const captchaConfig = JSON.parse(_.get(res, 'result.response.value'));
+        this.googleCaptchaSiteKey = captchaConfig.key || '';
+        this.isCaptchEnabled = captchaConfig.isEnabled || false;
+      }
+    }, (err: any) => {
+      this.toasterService.error(this.resourceService.frmelmnts.instn.t0056);
```

Comment:
For this no need to show toaster message

Reply:
should I remove this error handling part?